README.md

```diff
@@ -38,12 +38,22 @@ node guide.js
 
 After the server is successfully started, use a web browser to navigate to the GUIDE ITS Node.js server. The first person to sign in through the web admin UI will be given admin privileges.
 
-## Reset Database
+## Reset Database (Development)
 
 ```
 mongo mongodb://localhost/guide3 --eval "db.dropDatabase()"
 ```
 
+## Configure Database (Production)
+
+The following fields should be indexed for database performance:
+
+| Collection | Indexed Fields |
+| ----- | -------------- |
+| sessions | "id", "studentId" |
+| students | "id" |
+| conceptobservations | "studentId" |
```

Comment:
You might want to include the specific index syntax for each one.
For example: `{"id": 1}`.
The reason being that you can also do compound indexes where two fields are indexed together. But in these cases we only indexed the fields independently.
Also it looks like can use different values instead of `1` for the index. For example `-1` does something different.